compute/cker/include/cker/operation/RoPE.h

```diff
@@ -45,6 +45,9 @@ inline void RoPE(const RoPEMode mode, const Shape &input_shape, const T *input_d
   const int32_t i2_n = MatchingDim(input_shape, 2, output_shape, 2);
   const int32_t i3_n = MatchingDim(input_shape, 3, output_shape, 3);
 
+  if (i3_n % 2 != 0)
```

Comment:
I didn't follow this issue, so
could you explain why i3_n should be even number ?

Reply:
related : https://github.com/Samsung/ONE/pull/14176